api/api.proto

```diff
@@ -4464,6 +4476,8 @@ message SdkCloudBackupCreateRequest {
   // DeleteLocal indicates if local snap created for backup must be deleted after
   // the backup is complete
   bool delete_local = 7;
+  // Indicates if this is a backup restore for nearsync
+  bool near_sync_backup_restore = 8;
```

Comment:
Why do we need `restore` as part of this Backup create request? Are we trying to do a restore as part of the same backup request?

Reply:
So looks like we are overloading the BackupCreate request to perform a near-sync migration. Should we introduce a new API `CloudMigrateStart()` instead?